### PR TITLE
MX-15179: remove extra gasLimit field

### PR DESCRIPTION
**Pull request by dragos-rebegea**

### CODE REVIEW

**Review by sstanculeanu — APPROVED** [review]

File: clients/multiversx/client.go [hunk 1]
```diff
@@ -402,9 +402,8 @@ func (c *client) computeExtraGasForSCCallsBasic(batch *clients.TransferBatch) sc
 			continue
 		}
 
-		computedLen := 2                                                         // 2 extra arguments separators (@)
-		computedLen += len(deposit.Data) * 2                                     // the data is hexed, so, double the size
-		computedLen += len(big.NewInt(int64(deposit.ExtraGasLimit)).Bytes()) * 2 // the gas is converted to bytes, then hexed
+		computedLen := 2                     // 2 extra arguments separators (@)
```
Comment (iulianpascalau) [inline clients/multiversx/client.go:405]: should this be changed to 1?

File: clients/multiversx/client_test.go [hunk 2]
```diff
@@ -817,8 +813,7 @@ func TestClient_PerformAction(t *testing.T) {
 				for _, dt := range batch.Deposits {
 					dataStrings = append(dataStrings, depositToStrings(dt)...)
 					if len(dt.Data) > 0 {
-						gasLimitBytes := big.NewInt(int64(dt.ExtraGasLimit)).Bytes()
-						extraGas += (uint64(len(dt.Data)+len(gasLimitBytes))*2 + 2) * args.GasMapConfig.ScCallPerByte
+						extraGas += (uint64(len(dt.Data))*2 + 2) * args.GasMapConfig.ScCallPerByte
```
Comment (iulianpascalau) [inline clients/multiversx/client_test.go:816]: + 1 here ?

File: clients/multiversx/client.go (hunk 1 above)
Comment (dragos-rebegea) [reply clients/multiversx/client.go:405]: This will be changed in the next PR

File: clients/multiversx/client_test.go (hunk 2 above)
Comment (dragos-rebegea) [reply clients/multiversx/client_test.go:816]: This will be changed in the next PR

**Review by iulianpascalau — APPROVED** [review]

**Pull request merged**